internals/models.py

```diff
@@ -474,6 +480,13 @@ class Feature(DictModel):
 
   DEFAULT_CACHE_KEY = 'features'
 
+  def __init__(self, *args, **kwargs):
+    # Initialise Feature.blink_components with a default value
+    if 'blink_components' not in kwargs:
+      kwargs['blink_components'] = [BlinkComponent.DEFAULT_COMPONENT]
+
+    super(Feature, self).__init__(*args, **kwargs)
+
   @classmethod
```

Comment:
As mentioned [here](https://github.com/GoogleChrome/chromium-dashboard/pull/1351#discussion_r645485566), we could not pass the named argument `default=` while declaring the property `blink_components`. This is a workaround for that. It seems to work properly.